spec/features/articles_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe "Managing articles" do
+  describe "viewing the index page" do
+    let!(:post_1) { create :article }
+    let!(:post_2) { create :article }
+
+    it "shows all articles" do
+      visit '/articles'
+      expect(page).to have_content(post_1.title)
+      expect(page).to have_content(post_2.title)
+    end
+  end
+  
+  describe "creating an article" do
+    before do
+      visit '/articles'
+      click_link "New Article"
+    end
+
+    describe "missing some fields" do
```

Comment:
this is more of an opinion than a rule, but I would rather see `context` here instead of `describe`. Personally, I like to use `describe` to say what it is I'll be testing, then `context` to say any situational information.